src/components/Tiles/MovieTile.js

```diff
@@ -0,0 +1,19 @@
+import { render } from '../../shared/dom';
+
+export const MovieTile = ({ renderOn, movie }) => {
```

Comment:
Generally, the convention should be

For screens we are using the render function, for reusable component, we just need to return the plain html

that way we don't need to pass an ID (which would be painfull), but we would inject it in the html

Ex.

`<div> ${MovieTIle({...someArguments})} </div>` ~ now when MovieTile is returning the html template, it's being equal to just writing this html between these tags, and we have to write it that way, when we would inject like dynamic number of these components

Reply:
Up, this is not done still

Reply:
I fixed it now